src/plugins/gimbal/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 add_library(mavsdk_gimbal
     gimbal.cpp
     gimbal_impl.cpp
+    gimbal_manager.cpp
+    gimbal_manager_impl.cpp
+    gimbal_device.cpp
+    gimbal_device_impl.cpp
```

Comment:
I think these should be new plugins in a separate folder to follow the general plugin structure.

Also, from the naming, I would suggest this as the naming to avoid confusion:
- gimbal: includes the old gimbal protocol and the new gimbal protocol and uses the new one if the autopilot supports it).
- gimbal_device_tester: just for testing, not for normal usage.

Reply:
Got it, I'm going to do some refactoring and moving code around today